tutorials/introductory/usage.py

```diff
@@ -149,7 +149,7 @@
 #
 # So one can do (OO-style)
 
-x = np.linspace(0, 2, 100)
+x = np.linspace(0, 2, 100)  # This is data for plotting.
```

Comment:
what about # 100 values, equally spaced between 0 & 2

Reply:
IMHO that'd be one level too low. It describes the actual content of x, which is (more or less clearly) already stated by the code. But more importantly, the exact content is not particularly relevant. If we want a comment, it should state the purpose, hence my proposal above "sample data".

Reply:
I agree with Tim here.